files/telegraf_debug.conf

```diff
@@ -0,0 +1,59 @@
+[global_tags]
+[agent]
+  interval = "10s"
+  round_interval = true
+  metric_batch_size = 1000
+  metric_buffer_limit = 10000
+  collection_jitter = "1s"
+  flush_interval = "10s"
+  flush_jitter = "0s"
+  precision = "0s"
+  debug = false
+  quiet = false
+  hostname = "{{.Hostname}}"
```

Comment:
@mariana-dias-alves for retro-compatibility this should have a default value "telegraf", can you try to use an env var with a default value ?

Reply:
![image](https://github.com/MOV-AI/containers-telegraf/assets/139454338/b3fa05a4-96e0-4c1c-81b7-5b9bea39835c)